lib/zone.ts

```diff
@@ -587,43 +611,22 @@ interface EventTask extends Task {
   type: 'eventTask';
 }
 
-/**
- * Extend the Error with additional fields for rewritten stack frames
- */
-interface Error {
-  /**
-   * Stack trace where extra frames have been removed and zone names added.
-   */
-  zoneAwareStack?: string;
-
-  /**
-   * Original stack trace with no modifications
-   */
-  originalStack?: string;
-}
-
 /** @internal */
 type AmbientZone = Zone;
 /** @internal */
 type AmbientZoneDelegate = ZoneDelegate;
 
 const Zone: ZoneType = (function(global: any) {
```

Comment:
Just to clarify the interface and implementation and reduce the zone.ts size, it is a little big, user may easily check the API from the interface file.

But we have the zone.js.d.ts file, so I think it is also ok in current way.